.github/workflows/coverity.yaml

```diff
@@ -0,0 +1,48 @@
+name: Coverity
+on:
+#  schedule:
+#    # run daily 20:00 on master branch
+#    - cron: '0 20 * * *'
+  push:
+    branches:
+      - coverity_scan
+jobs:
+  coverity:
+    name: Coverity
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [12.3]
+        os: [ubuntu-18.04]
```

Comment:
Nit: I don't think we'll need to run this on different OS:es.

Reply:
we don't but since the os is used as cache key might as well have it here and use it as variable